config/ucx.conf

```diff
@@ -35,3 +35,7 @@ UCX_GDR_COPY_LAT=get:1.65e-6,put:0.65e-6
 [AMD Genoa]
 CPU model=Genoa
 UCX_DISTANCE_BW=auto,sys:5100MBs
+
+[AMD Turin]
```

Comment:
Is that possible to set `UCX_DISTANCE_BW` one time for all AMD CPU families? Something like:

```
[AMD]
CPU vendor=AMD
UCX_DISTANCE_BW=auto,sys:5100MBs
```

Reply:
Could you please explain more about  "UCX_DISTANCE_BW" ? Why we have to keep 'sys' at 5100MBs?[I was not sure about this, I used the same value that used in Rome, Milan & Genoa] Is this should be calculated at runtime?

Reply:
That's a good question, that's pretty old setting that was measured long time ago on some AMD system and I am not sure whether we need to set it for Turin systems too. @yosefe WDYT? 

My previous comment was only suggesting to set it one time for all AMD system instead of patching each CPU generation separately, but probably if we don't want to set this value for this generation this change would redundant too.

Reply:
@yosefe could you please comment on "UCX_DISTANCE_BW" ?

Reply:
UCX_DISTANCE_BW is specifying the expected PCIe bandwidth as a function of distance as defined here: https://github.com/openucx/ucx/blob/master/src/ucs/config/global_opts.c#L78

Reply:
1. it is representing the expected PCIe badnwidth in case the devices are on different NUMA nodes
2. If not provided a default value is used. we can define the config setting as AUTO, and the code will calculate the BW if the value is auto.
3. it's between devices on different NUMA node, traversing system interconnect (such as QPI). For example GPU and RDMA NIC.

Reply:
Currently UCX cannot determine the actual ucs_global_opts.dist.sys.bandwidth value, the default is [220 MB/s](https://github.com/openucx/ucx/blob/c4acc6c3f/src/ucs/sys/topo/base/topo.c#L439) which is probably not right value for AMD.

Reply:
Can it be derrived reading /proc or /sys entries ? How shall we proceed further here?

Reply:
> Can it be derrived reading /proc or /sys entries ? How shall we proceed further here?

This is a question for AMD, I guess. I'm not aware of such way, but would be great to have.
I'd suggest for now to define single BW value for all AMD CPU's , as suggest by @ivankochin

Reply:
@yosefe @ivankochin Apologies for my ignorance, but I want to ensure I understand this correctly before proceeding. With 2 Turin-only nodes connected via IB, are there benchmark tests available to measure the impact of varying "UCX_DISTANCE_BW=auto,sys:_values_from_low_to_high_" values? Or Does this only apply to GPU-to-RDMA transfers in a GPU cluster?

Reply:
This setting may affect the threshold switching of GPU protocols, so probably benchmarks like "osu_bw D D" could uncover  the difference on some systems.